packages/replay-next/src/suspense/TimeoutCache.ts

```diff
@@ -0,0 +1,23 @@
+import { createStreamingCache } from "suspense";
+
+export const TimeoutCache = createStreamingCache<
```

Comment:
q: what's the lifetime of this cached value? at what times it's created and disposed?

Reply:
Lifetime is forever (in memory) rather than being tied to a specific component. It's created when the React DevTools panel renders for the first time (and tries to fetch annotations). Both of those are intentional in this case, since a user might switch away to the Viewer tab (or some other Devtools tab) and come back.

Reply:
ah, ok - so the signal here is handled because it's the right thing to do but this one won't actually ever be aborted because that requires an explicit `.abort(...params)` call. got it 👍

Reply:
Yeah. This specific one doesn't receive an abort bc it's only used in a suspense boundary so it will go away on its own once the annotations data has loaded. The only things further down in the tree that can suspend (inspected elements panel) have lower-level suspend boundaries that catch them.

That being said, we could try doing an explicit abort here. It's a little weird bc the two sites are separated a bit, but if we wanted to we could probably use the `useCacheStatus` hook to listen for annotation loading status and then abort in an effect.

Reply:
yeeeah, I wouldn't bother since this is just a timeout and we can safely ignore it switching the state after the annotations already resolved (and resumed the suspended tree)